api/challenge_api/app/channels/application_cable/channel.rb

```diff
@@ -0,0 +1,4 @@
+module ApplicationCable
+  class Channel < ActionCable::Channel::Base
```

Comment:
My Rails knowledge is rusty :), what does this do? `Channel < ActionCable::Channel::Base` I'm assuming we're setting up the class structure with inheritance?

Reply:
My mistake-- I think that this is part of ActionCable too, so I'll take it out